vllm/model_executor/model_loader/loader.py

```diff
@@ -99,13 +99,34 @@ def _initialize_model(vllm_config: VllmConfig, prefix: str = "") -> nn.Module:
         param.name for param in signatures.parameters.values()
         if param.kind == inspect.Parameter.KEYWORD_ONLY
```

Comment:
Can we allow `POSITIONAL_OR_KEYWORD` arguments as well?

Reply:
relaxed the constraint to only `vllm_config` and `prefix` . does it sound good to you?